src/routes/RequestsRoute.js

```diff
@@ -354,7 +354,6 @@ class RequestsRoute extends React.Component {
 
   componentDidMount() {
     this.setCurrentServicePointId();
-    this.buildRecordsForHoldsShelfReport();
```

Comment:
The body of the `setCurrentServicePointId` function contains an explicit call to the `buildRecordsForHoldsShelfReport` function.